### ISSUE TITLE
Latest windows installers from dungeon are not working

### ISSUE THREAD

**Issue by djallado** [learningequality/installers#64]
**Branch: Develop**

**Test Latest Installers from dungeon** 
    1. KALiteSetup-0.14.dev11.exe  	Jun 24, 2015 10:03:52 AM
    2. KALiteSetup-0.14.dev14.exe  	Jun 24, 2015 10:04:01 AM

**Expected behavior:**
After you click the `start server` menu, it notify you like `the server is....` screenshot below: 
![4](https://cloud.githubusercontent.com/assets/8664071/8355122/eb6aae6a-1b7f-11e5-867c-2cfc8a6579ea.png)
And after waiting for a few seconds another notification will appear like `The server will be...` screenshot below:
![5](https://cloud.githubusercontent.com/assets/8664071/8355154/1900f1c2-1b80-11e5-9167-f131527c878f.png)
After that you can select `load in browser` from the menu and set Super user account like screenshot below:
![createsuperuser](https://cloud.githubusercontent.com/assets/8664071/8355245/aa28024e-1b80-11e5-86d0-7262a70adb3f.png)

**Actual behavior:**
After you click the `start server` menu, it notify you like:
![4](https://cloud.githubusercontent.com/assets/8664071/8355276/d8e141b8-1b80-11e5-8f2a-ddd41857ac53.png)
After waiting for a few seconds no other notification will appear.
You can't select the `load in browser` menu. The menu is disabled like screenshot below:
![12](https://cloud.githubusercontent.com/assets/8664071/8355388/95df23de-1b81-11e5-85d0-f87adbf51d2b.png)

**Comment by aronasorman** [comment]
Thanks @djallado --  it happens when building the windows installer with `wine`, as @EdDixon managed to replicate it.

**Comment by amodia** [comment]
I also verified the latest build #152 from dungeon and I observe that the user account I set during installation can't login. I was able to login using the account I set in pop up dialog create superuser in browser. 

Here's the screenshots.
![screen shot 2015-06-26 at 4 04 49 pm](https://cloud.githubusercontent.com/assets/8663934/8379955/ce85c312-1c55-11e5-93a9-485be067491a.png)

![screen shot 2015-06-26 at 10 56 46 pm](https://cloud.githubusercontent.com/assets/8663934/8380124/ad0a1a7a-1c56-11e5-8fb8-7d8a08d04d1a.png)
